data/src/main/java/com/example/data/service/response/OneCallResponse.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.data.service.response
+
+data class OneCallResponse(
+    val lat: Double,
+    val lon: Double,
+    val daily: List<OneCallDailyResponse>
+)
+
+data class OneCallDailyResponse(
+    val dt: Long,
+    val temp: OneCallTempResponse,
+    val weather: List<OneCallWeatherResponse>,
+    val feels_like: OneCallFeelsLikeResponse,
```

Comment:
Kotlin/Java classes shouldn't have attributes with underscores, rename to "feelsLike" and use `@SerializedName(value = "feels_like")` for Gson to find the attribute.